prisma/schema.prisma

```diff
@@ -164,17 +165,44 @@ model Centre {
     name            String @unique
     displayText     String
     displaySequence Int
-    streetAddress   String
-    city            String
-    stateProvince   String
-    country         String
+    streetAddress   String?
+    city            String?
+    stateProvince   String?
+    country         String?
 
+    members Member[]
```

Comment:
food for thought: do we need a separate page to manage the center's members? If yes I'm thinking about tying the members using groups so that we can reuse the member group management functionality.Let's discuss this on our Sunday's call.